docs/src/design-system/charts/generate-data.tsx

```diff
@@ -0,0 +1,32 @@
+import Perlin from '../../utils/perlin';
+
+const generateData = () => {
+  const labels = Array(24 * 60)
+    .fill(0)
+    .map((_, i) => {
+      return ('0' + ~~(i / 60) + ': 0' + Math.round(60 * ((i / 60) % 1))).replace(
```

Comment:
To the problem @EdAdaptive has pointed out about the extra space in front of minutes on the x axis

It's such a small fix I don't know if it's worth putting in a separate MR